guides/gatherers.md

```diff
@@ -203,6 +204,65 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/packageversion_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion_test.go)
 
+### saphostctrl
+
+This gatherer allows access to certain webmethods that `saphostctrl` implements. An argument is required to specify
+which webmethod should be called. This webmethod is passed to the `saphostctrl` command-line tool thought the `-function` argument. Note that not all webmethods are supported.
+
+Supported webmethods:
+  -  `Ping`
+  -  `ListInstances`
+
+A `Ping` call with a successful return should look like:
+
+```
+{
+  "status": "SUCCESS", 
```

Comment:
For consistency, we should follow the rust format, as we have in sbddump:
```rust
{
  "status" => "SUCCESS",
  "elapsed" => 543341,
},
```
Or we change the other to follow this format. I'm not sure what is the format used by rhai